src/main/resources/run-class.sh

```diff
@@ -51,6 +54,15 @@ stop)
   exit 1
 esac
 
+# Check for any additional args to pass through to the main java class, separated by "--".
```

Comment:
You're really checking for a `--` and stripping it, giving an error if it's missing.